src/cooperative.jl

```diff
@@ -359,8 +361,8 @@ function add_EC_peak_summary!(
 
     # get main parameters
     market_data = ECModel.market_data
-    peak_categories = profile(market_data, "peak_categories")
-
+    peak_categories = market_profile_by_user(ECModel,"user1","peak_categories")
```

Comment:
Better to add a time series into gen data and load that one directly

Reply:
Resolved